api/api/services/file_storage.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import os
+from typing import override
+
+from core.storage.file_storage import FileData, FileStorage
+
+
+def _default_file_storage() -> FileStorage:
+    # We return an empty blob storage if the env variables are not set
+
+    connection_string = os.getenv("WORKFLOWAI_STORAGE_CONNECTION_STRING", "")
```

Comment:
nit: use a `WORKFLOWAI_STORAGE_KIND` to make choices more explicit ?

Reply:
That's what the scheme the connection string is for :) 
We already have a LOT of environment variables...